src/components/DataMessenger/DataMessenger.js

```diff
@@ -251,7 +267,7 @@ export class DataMessenger extends React.Component {
 
       clearTimeout(this.windowResizeTimer)
       clearTimeout(this.executeQueryTimeout)
-    } catch (error) {}
+    } catch (error) { }
```

Comment:
maybe add a console.error ?

Reply:
I think since this is a clean up function there isn't really a need to console the error as it won't give useful information